whipper/test/test_common_config.py

```diff
@@ -66,3 +66,28 @@ def testDefeatsCache(self):
         defeats = self._config.getDefeatsCache(
             'PLEXTOR ', 'DVDR   PX-L890SA', '1.05')
         self.assertEquals(defeats, True)
+
+    def test_get_musicbrainz_server(self):
+        # Test default return value
+        self.assertEquals(self._config.get_musicbrainz_server(),
+                          'musicbrainz.org')
+
+        # Section setup
+        self._config._parser.add_section('musicbrainz')
+
+        # Test normal URL
+        self._config._parser.set('musicbrainz', 'server',
+                                 '192.168.2.141:5000')
+        self._config.write()
+        self.assertEquals(self._config.get_musicbrainz_server(),
+                          '192.168.2.141:5000')
```

Comment:
I think it would be worth adding a msg= param to assertEquals along the lines of "Precondition check: config not set to expected value of 192...".